### ISSUE TITLE
PhotoTaken event?

### ISSUE THREAD

**Issue by CalebPourchot** [Apparence-io/CamerAwesome#411]
I've searched and searched and can't find anything in the documentation or google about how to be told that a photo was taken successfully.

I see this in the source code of photo_camera_state.dart:
```
@override
  CaptureMode get captureMode => CaptureMode.photo;

  /// Photos taken are in JPEG format. [filePath] must end with .jpg
  ///
  /// You can listen to [cameraSetup.mediaCaptureStream] to get updates
  /// of the photo capture (capturing, success/failure)
```
I don't think I can get to cameraSetup.mediaCaptureStream as a consumer of CameraAwesomeBuilder, right? What am I missing? Seems like this is a pretty typical use case?

**Comment by g-apparence** [comment]
Hi we recently merged it on the main branch.
You can now add a callback to the "takePicture" function (See #401 ).
As this is only available for the 1.4.0 branch for now I have to do the same on the 2.0.0. 
This is one of the latest things to do before releasing the 2.0.0 version.

**Comment by CalebPourchot** [comment]
Fabulous! Thanks so much for this plug-in. I'm looking forward to using it.
